src/current/advisories/a190483.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Technical Advisory 190483
+advisory: A-190483
+summary: Queries reading a table that uses at least one explicit column family in its definition and was restored from a backup using version v22.2.7 to v22.2.13 or v23.1.0 to v23.1.8 may return errors or incorrect results.
+toc: true
+affected_versions: v22.2.7 to v22.2.13, v23.1.0 to v23.1.8
+advisory_date: 2023-09-21
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+Queries reading a table that uses at least one explicit column family in its definition and was [restored](https://www.cockroachlabs.com/docs/stable/backup-and-restore-overview) from a backup using the following versions may return errors or incorrect results:
+
+- v22.2.7 to v22.2.13  
+- v23.1.0 to v23.1.8
+
+Restored tables that do not use explicit column families, or that were restored using versions prior to v22.2.7, after v22.2.14, or v23.1.9, are unaffected.
```

Comment:
> prior to v22.2.7, after v22.2.14, or v23.1.9,
The status of 23.1.9 seems a _little_ unclear in this phrasing? Under some readings I wonder if it might appear that 23.1.9 is an affected version.

Reply:
```suggestion
Restored tables that do not use explicit column families, or that were restored using versions prior to v22.2.7,  v22.2.14 or later, or v23.1.9 or later, are unaffected.
```